rust/gui-client/src-tauri/src/client/gui/system_tray.rs

```diff
@@ -277,21 +328,12 @@ mod tests {
     }
 
     #[test]
-    fn dns_resource_with_url() {
```

Comment:
Got rid of this by just testing the URL marker arrows (`<https://example.com>`) in every unit test